types/polka/index.d.ts

```diff
@@ -95,6 +95,13 @@ declare namespace polka {
          * It receives all requests and tries to match the incoming URL against known routes.
          */
         handler(req: Request, res: ServerResponse, parsed?: Url): void;
+
+        /**
+         * The instantiated `server` Polka creates when `listen()` is called.
+         * `server` is only created if a server was not provided via `option.server`
+         * `server` will be undefined until polka.listen is invoked or if a server was provided.
+         */
+        server?: Server;
```

Comment:
In the docs https://github.com/lukeed/polka I do not see `server` to be exposed?

Reply:
`server` is exposed in the code snippet in the code example provided [here](https://github.com/lukeed/polka#listen) in the polka README.

From Polka README:

```javascript
// README.md
const { server, handler } = polka().listen();

// Or this!
const app = polka().listen(PORT, onAppStart);

app.use('users', require('./users'))
  .get('/', (req, res) => {
    res.end('Pretty cool!');
  });
```

`server` also exists on the Polka class as shown [here](https://github.com/lukeed/polka/blob/master/packages/polka/index.js#L33)

Reply:
Makes sense! Thanks! :)

Reply:
No problem, thanks for the approval 😊